src/Engine.php

```diff
@@ -143,7 +143,9 @@ private function runInitialTestSuite(): void
             $initialTestSuitProcess->getOutput()
         );
 
-        $this->memoryLimitApplier->applyMemoryLimitFromProcess($initialTestSuitProcess, $this->adapter);
+        // Limit the memory used for the mutation processes based on the memory used for the initial
+        // test run
```

Comment:
don't think it should be on the second line

Reply:
trying to stick to 100